pipeline/pipelines/core_generation_pipeline.py

```diff
@@ -71,9 +71,24 @@ def _get_core_codegen_tasks(self, **kwargs):
         return [protoc_tasks.ProtoCodeGenTask]
 
 
+class _JavaCoreTaskFactory(CoreTaskFactoryBase):
+    """Responsible for the protobuf flow for Java language.
+
+    TODO(shinfan): Add packaging tasks
+    """
+
+    def _get_core_codegen_tasks(self, **kwargs):
+        return [protoc_tasks.ProtoCodeGenTask,
+                protoc_tasks.JavaCoreProtoCopyTask]
+
+    def get_validate_kwargs(self):
+        return ['gapic_api_yaml'] + code_gen.COMMON_REQUIRED
```

Comment:
I am not sure what this field does and gapic_api_yaml is a field in the artman_core config: https://github.com/googleapis/googleapis/blob/master/gapic/core/artman_core.yaml#L15

I can remove it if it is not necessary.